native_client/dotnet/DeepSpeechClient/DeepSpeech.cs

```diff
@@ -204,14 +204,11 @@ public unsafe void PrintVersions()
         /// <summary>
         /// Creates a new streaming inference state.
         /// </summary>
-        /// <param name="aPreAllocFrames">Number of timestep frames to reserve.
-        /// One timestep is equivalent to two window lengths(20ms).
-        /// If set to 0 we reserve enough frames for 3 seconds of audio(150).</param>
         /// <param name="aSampleRate">The sample-rate of the audio signal</param>
         /// <exception cref="ArgumentException">Thrown when the native binary failed to initialize the streaming mode.</exception>
-        public unsafe void SetupStream(uint aPreAllocFrames, uint aSampleRate)
```

Comment:
@lissyx 

https://github.com/mozilla/DeepSpeech/blob/b92cea1c0b38bb94f25585a4c8f970287a37caeb/examples/net_framework/DeepSpeechWPF/MainWindow.xaml.cs#L255 now only needs the sample rate, will fail with the extra 0

Reply:
We don't build that? I thought we did, I checked twice for SetupStream calls 😕

Reply:
Good catch anyway, thanks

Reply:
@lissyx 

> 
> 
> We don't build that? I thought we did, I checked twice for SetupStream calls 😕
We don't build the WPF example : 
https://github.com/mozilla/DeepSpeech/blob/69ad6016a7e991138959f079c4067769820abacf/taskcluster/tc-tests-utils.sh#L879-L916

I will try to add a window on my time and add the example to the builds, should be straightforward.